### ISSUE TITLE
New AstropyDeprecationWarning issued by SunPy when using astropy 3.0

### ISSUE THREAD

**Issue by DanRyanIrish** [sunpy/sunpy#2473]
When SunPy inherits from astropy 3.0, the following ```AstropyDeprecationWarning``` is issued:
```
WARNING: AstropyDeprecationWarning: 
The 'recommended_units' attribute is deprecated since 3.0 and may be removed
in a future version. Its main use, of representing angles in degrees in frames,
is now done automatically in frames. Further overrides are discouraged but can
be done using a frame's ``frame_specific_representation_info``.
 [astropy.coordinates.representation]
```
This should be addressed or we should at least be aware of its consequences.

**Comment by nabobalis** [comment]
We have alot of those.

**Comment by Cadair** [comment]
So I think this is fixed in master as we removed the whole `sunpy.coordinates.representation` module. For 0.8 we should explicitly catch this warning on import of reprentation and scilence it, as it's just annoying for the users.

**Comment by yashkgp** [comment]
I would like to work on this.

**Comment by Cadair** [comment]
@nabobalis yes, we should fix all of them where possible. This issue is about this one, a list of the others would probably be useful.

@yashkgp we should be able to just catch it once inside the `sunpy.coordinates.representation` hopfully.

**Comment by nabobalis** [comment]
Fixed by #2476
